### DISCUSSION TITLE
Welcome to CustomTkinter Discussions!

### DISCUSSION THREAD

**Discussion by TomSchimansky** [TomSchimansky/CustomTkinter#11] [Announcements]
## Welcome to CustomTkinter Discussions!

Here you can discuss questions, showcases and ideas related to CustomTkinter. For reporting bugs and propose new features, and only for this, use the issues tab instead.

**Comment by elads11** [comment]
Hey, first of all, great library!
I have a small app for work, and I wanted to stylish it more, so that's how I've come upon this library.
The problem I'm facing is when I'm converting a button for example in my code to your CTKinter button, I get exceptions when I click on it.
That's because it doesn't recognize the state=disabled, which works perfectly with the regular TK button.

**Reply by TomSchimansky** [reply]
Can you show me the code for creating the button and setting it to disabled state?

**Reply by TomSchimansky** [reply]
Saw it was already resolved...

**Comment by NoahBarrall** [comment]
Hey man! Great library, I'm pretty new to it, so I was just wondering if there was any way to display user input on the screen? Like for example, I take in a task a user wants to do and I display it in a box or something on the screen?? Thanks!!

**Comment by Priestly-flutter** [comment]
Good day, please can I have the documentation for customtkinter in a pdf format?

**Comment by MaxNiark** [comment]
HERE for visual rendering with mouse motion bind

1ms display updating

![image](https://user-images.githubusercontent.com/126392328/235452045-57647642-8906-43dc-9805-14b81e02e665.png)
![image](https://user-images.githubusercontent.com/126392328/235451721-abb77260-f4f3-459d-afe6-41b6f60baeee.png)
![image](https://user-images.githubusercontent.com/126392328/235451901-4ff17aaf-804c-4478-8f93-bf62be56eb9c.png)

**Comment by ghost** [comment]
@TomSchimansky  - Maybe close this discussion/lock it to only maintainers?

**Comment by NelDavi** [comment]
Is it possible to create a treeview with CustomTkinter? If so, how do you do it?

**Reply by ghost** [reply]
I think it's possible. You can use CTkLabels for any text elements, using the grid or padding you can align those labels however you want. Maybe use transparent buttons to hold images of arrows.

**Comment by Vannak4273** [comment]
Can we change the custom cursor whenever we hover any widgets? It seems like it has been removed this feature unlike tk or tkk

**Reply by ghost** [reply]
I would suggest making a new, feature-request issue.
